### ISSUE TITLE
Crude Oil Dupe/bug

### ISSUE THREAD

**Issue by StoobyMcDooby** [FTBTeam/FTB-Continuum#108]
* **Modpack Version**: 

* **Issue**:  Forge 14.23.3.2702, Minecraft 1.12.2, Continuum pack version 1.1.0

* **Link to Log or Crash File Paste**:  Not a crash

* **Is it Repeatable?**: Yes

* **Mod/s Affected**:  Not sure

* **Known Fix**:  Don't abuse

Issue:  Built a distillation tower from Immersive Petroleum.  Doing regular progression, built a pumpjack to supply oil to it.  Got 4 cans of crude oil from a quest reward somewhere along the lines from a quest, went and put the 4 cans of crude oil into the top left slot of the distillation tower (input slot) and the cans stay there, never emptying.  They do, however, fill the internal liquid tank of the distillation tower with crude oil (infinite crude oil).  Looked at it for 3 seconds and went, "Well that ain't right..."

**Comment by TNovalis** [comment]
This is still current on **version 1.1.0** of the pack.

I have been able to repeat it on a different server. It seems that Immersive Petroleum and the Forestry cans are not compatible with one another, so it is unable to empty the tin can and use the oil, instead it knows there is a compatible crude oil in the can but will continue and attempt to empty the can.

TL;DR: Works on **1.1.0** and they are not fully compatible with one another.

I can post a video if needed.

**Comment by aaronhowser1** [comment]
Post this on the [Immersive Petroleum](https://github.com/Flaxbeard/ImmersivePetroleum/issues) issue tracker as well

**Comment by phit** [comment]
this is still and issue on both latest revelation (2.1.0) and latest continuum (1.3.1)

**Comment by tfox83** [comment]
this has been reported and the fix will go in when it becomes avail

**Comment by phit** [comment]
@tfox83 where? can't find an issue on their tracker

**Comment by tfox83** [comment]
Closing as waiting mod for a few months.  If a fix is released I will get in next update

**Comment by Armarius** [comment]
As workaround add a shapless receipe to convert _<forge.buchetfilled>.withTag({FluidName: "crude_oil", Amount: 1000})_ to _<forge.buchetfilled>.withTag({FluidName: "oil", Amount: 1000})_ and vise versa. Hope that helps a little bit.
